tinygrad/helpers.py

```diff
@@ -79,31 +79,32 @@ def __init__(self, priority, itemsize, name, np, shape):
   def __repr__(self): return f"dtypes.{self.name}({self.shape})"
 
 class dtypes:
-  @staticmethod # static methds on top, or bool in the type info will refer to dtypes.bool
-  def is_int(x: DType)-> bool: return x in (dtypes.int8, dtypes.uint8, dtypes.int32, dtypes.int64)
-  @staticmethod
-  def is_float(x: DType) -> bool: return x in (dtypes.float16, dtypes.float32, dtypes.float64, dtypes._half4, dtypes._float4)
-  @staticmethod
-  def is_unsigned(x: DType) -> bool: return x in (dtypes.uint8, dtypes.uint32, dtypes.uint64)
```

Comment:
`is_int` used to be wrong? Checking for signed types + `uint8`.  Used in `llvmir.py` to check if type is integer, which I also believe to be a bug (since `is_int` does not include all int types). Hence why I changed from `is_int`, `is_unsigned_int` and `is_signed_int`